package-lock.json

```diff
@@ -20834,20 +20834,15 @@
 			}
 		},
 		"sodium-native": {
-			"version":
-				"github:LiskHQ/sodium-native#dd0319f679cc5b4821c9c409fa02884a2c80938e",
-			"from": "github:LiskHQ/sodium-native#dd0319f",
+			"version": "2.2.4",
+			"resolved":
+				"https://registry.npmjs.org/sodium-native/-/sodium-native-2.2.4.tgz",
+			"integrity":
+				"sha512-zE3lJAEN9R/XzJmNUqfyqL3vAnES9rFuyeq5ouHmCOdkVcY5UKbCcl7eUyZ+LG4RcqVfx8CAcgwv9HRpgoNrlg==",
 			"requires": {
-				"ini": "1.3.5",
-				"nan": "2.10.0",
-				"node-gyp-build": "3.0.0"
-			},
-			"dependencies": {
-				"nan": {
-					"version": "2.10.0",
-					"resolved": "https://registry.npmjs.org/nan/-/nan-2.10.0.tgz",
-					"integrity": "sha1-ltDNYQ69WNS03pzAxoKM2pnHVI8="
-				}
+				"ini": "^1.3.5",
```

Comment:
These carets scare me. @fchavant Can we test if they are just being ignored when using `npm ci`?

Reply:
That's the difference due to the dependencies freeze on LiskHQ fork. As @diego-G points out, we should have the strict nested dependencies here.

Reply:
@MaciejBaj we were using `npm install` then tough; I'll test it out as @diego-G suggested